test/integration/assorted.js

```diff
@@ -164,18 +164,30 @@ test('selects .cjs test files', t => {
 	});
 });
 
-test('refuses to load .mjs test files', t => {
+test('load .mjs test files (when node supports it)', t => {
 	execCli('mjs.mjs', (err, stdout) => {
-		t.ok(err);
-		t.match(stdout, /AVA cannot yet load ESM files/);
-		t.end();
+		if (Number.parseFloat(process.version.slice(1)) >= 13 && process.platform !== 'win32') {
```

Comment:
We should figure out why it's failing on Windows. It can't be a Node.js issue.

Reply:
on windows a ERR_UNSUPPORTED_ESM_URL_SCHEME is thrown maybe drive letters or backward slashes have to be removed